GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -329,10 +329,10 @@ extension CameraViewController {
     
     fileprivate func configurePopupViewForUnsupportedQR(_ newQRCodePopup: QRCodeDetectedPopupView,
                                                         dismissCompletion: @escaping () -> Void) {
-        newQRCodePopup.qrText.textColor = .red
+        newQRCodePopup.qrText.textColor = giniConfiguration.unsupportedQrCodePopupColor
```

Comment:
In the `QRCodeDetectedPopupView` we are using a `GiniColor` and our `UIColor.from()` extension:
```
qrText.textColor = UIColor.from(giniColor: giniConfiguration.qrCodePopupTextColor)
``` 

We should do it the same here as well and then also rename it to `unsupportedQrCodePopupTextColor`.

```suggestion
        newQRCodePopup.qrText.textColor = UIColor.from(giniColor: giniConfiguration.unsupportedQrCodePopupTextColor)
```